### PR TITLE
tests: split travis spread execution in 2 jobs for ubuntu and non ubuntu systems

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This is the same change than #6631 which was deleted.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,7 +56,19 @@ jobs:
         # override the default install for language:go
         - true
       script:
-        - ./run-checks --spread
+        - ./run-checks --spread-ubuntu
+    - stage: integration
+      name: spread on other systems than ubuntu
```
Comment (bboozzoo) [inline .travis.yml]: And this could be named `Fedora, CentOS, OpenSUSE, Debian, Arch`

File: .travis.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
       script:
         - ./tests/lib/cla_check.py
     - stage: integration
-      name: spread
+      name: spread on ubuntu systems
```
Comment (bboozzoo) [inline .travis.yml]: Can we name it `Ubuntu 14.04, 16.04, 18.04, 18.10`

**Comment by bboozzoo** [comment]
Let's discuss the plan at the standup.

File: .travis.yml (hunk 1 above)
Comment (sergiocazzolato) [reply .travis.yml]: thanks for reviewing! It is ok for me, lets discuss the names on the standup

**Pull request merged**